pkg/controllers/uiplugin/compatibility_matrix_test.go

```diff
@@ -185,6 +185,26 @@ func TestLookupImageAndFeatures(t *testing.T) {
 			expectedKey:    "ui-troubleshooting-panel",
 			expectedErr:    nil,
 		},
+		{
+			pluginType:     uiv1alpha1.TypeMonitoring,
+			clusterVersion: "v4.16",
+			expectedKey:    "ui-monitoring",
+			expectedErr:    fmt.Errorf("plugin %q: no compatible image found for cluster version %q", uiv1alpha1.TypeMonitoring, "v4.16"),
+		},
+		{
+			pluginType:       uiv1alpha1.TypeMonitoring,
+			clusterVersion:   "v4.17",
+			expectedKey:      "ui-monitoring",
+			expectedFeatures: []string{"acm"},
```

Comment:
maybe this is a collection of more specific features like `["alerting", "metrics"]`, WDYT?

Reply:
As per our discussion, this has been updated to "acm-alerting" in the most recent commit.